data/tudortmund/tudo-apc-2016.csv

```diff
@@ -0,0 +1,12 @@
+institution,period,euro,doi,is_hybrid,publisher,journal,issn 
+Technische Universität Dortmund,2016,"952,00",10.17645/si.v4i3.546,FALSE,Cogitatio,Social Inclusion,2183-2803
+Technische Universität Dortmund,2016,"143,66",10.1107/S2414314616006970,FALSE,International Union of Crystallography,IUCR Data,2414-3146
+Technische Universität Dortmund,2016,"2232,50",10.1364/OE.24.012442,FALSE,OSA Optical Society of America,Optics Express,1094-4087
+Technische Universität Dortmund,2016,"2000,00",10.1002/open.201600048,FALSE,Wiley ,ChemistryOpen,2191-1363
+Technische Universität Dortmund,2016,"1617,93",10.1155/2016/9605450,FALSE,Hindawi,Advances in Materials Science and Engineering,1687-8434
+Technische Universität Dortmund,2016,"1713,60",10.1002/open.201600092,FALSE,Wiley ,ChemistryOpen,2191-1363
+Technische Universität Dortmund,2016,"400,00",,FALSE,sub\urban e.V.,suburban : zeitschrift für kritische stadtforschung,2197-2567
```

Comment:
@hagbeck These 3 articles do not have a DOI assigned, we need a link to the article fulltext to correctly identify them. You should add an "url" column to provide it, see [here](https://github.com/OpenAPC/openapc-de/wiki/Handreichung-Dateneingabe#optionale-felder) for details.